content/blog/_global/contact.md

```diff
@@ -0,0 +1,33 @@
++++
```

Comment:
Do we need a contact form?

Reply:
It can be used as a comment form. That is if we create an email for it that can accept spam :D

Reply:
Ok let's go without this for now ;)